ports/qwt/vcpkg.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "qwt",
+  "version": "6.2.0",
```

```suggestion
  "version-semver": "6.2.0",
```
This should be `version-semver`.